### ISSUE TITLE
key_press missing escape?

### ISSUE THREAD

**Issue by mrtodd** [oooutlk/tcltk#9]
TkKey::Escape is not included in the Enum.  Is there a reason or just an oversight?

Is there a way to bind any key press instead of requiring a specific key?

I noticed that "button_press" is defined to pick up any button.  There is a "button_pelease" defined as well (guessing there is a typo with that one).

**Comment by oooutlk** [comment]
Oops yes, missing escape, no bind to any key press and `button_pelease()` are oversights. Thanks for mentioning!

I am planning to fix this issue in next coming 24 hours. A new function `any_key_press()` will be added.

**Comment by mrtodd** [comment]
I also noticed that none of the numeric keypad keys are available (KP_Enter, KP_Add, KP_5, etc).  I guess having 'any_key_press()' should allow one to access any of the keys which were overlooked.

Thanks for looking at this so quickly.
-- Todd

    On Monday, November 20, 2023 at 12:05:24 AM GMT+8, oooutlk ***@***.***> wrote:  
 
 


Oops yes, missing escape, no bind to any key press and button_pelease() are oversights. Thanks for mentioning!

I am planning to fix this issue in next coming 24 hours. A new function any_key_press() will be added.

**Comment by oooutlk** [comment]
This issue has been fixed in [tk 0.1.7](https://crates.io/crates/tk/0.1.7).
I really need some responses from crate users to improve it and, thanks! 
The following is demonstrating `any_key_press()` usage:

```
use tcl::*;
use tk::*;

fn main() -> TkResult<()> {
    let tk = make_tk!()?;
    tk  .root()
        .bind( event::any_key_press(),
            tclosure!( tk, args: "%k", |keycode: std::ffi::c_int| -> TkResult<()> {
                println!( "You are pressing {keycode}" );
                Ok(())
            }))?;

    Ok(main_loop())
}
```

If you want to check text rather than key code, use `tclosure!( tk, args: "%A", |keycode: String| -> TkResult<()>` instead.
